### ISSUE TITLE
Baselines for DSO

### ISSUE THREAD

**Issue by narve** [euniversal/umei-api-specification#15]
DSOs want to have the possibility to fetch baselines for a given grid node. The baselines should presumably be an aggregated value based on portfolios on that grid node.

**Comment by cdmNSIDE** [comment]
Hello Narve,

Is there a need to change the API for that ?
I think all those scenarios could be handled by the access rights of the different market participants, isn't it ?

**Comment by narve** [comment]
Some of this can be handled by access rights, yes. However, fetching baselines for a given grid node is not part of the api for now. See issue #18 . Adding an optional parameter "gridNodeId" on /gridnodes would be sufficient as a minimum, I think.

**Comment by cdmNSIDE** [comment]
> Adding an optional parameter "gridNodeId" on /gridnodes would be sufficient as a minimum

I do not understand. Could you please explain ?
I think we can simply add a filter in the **GET /BaselinesIntervals** request to filter them by gridNodeId, isn't it ?
I think we may also add a filter there based on the portfolioId

**Comment by narve** [comment]
yes, sorry, meant filter on GET /BaselineIntervals like you said
